README.md

```diff
@@ -45,6 +45,14 @@ $ source venv/bin/activate
 (venv) $ python -m signal_media_exporter -h
 ```
 
+And to install for use from outside the project's directory:
+
+```
+$ poetry build
+$ python3 -m pip install dist/*.tar.gz
+$ signal_media_exporter -h
```

Comment:
```suggestion
$ signal-media-exporter -h
```
While the python package is named signal_media_exporter, the binary/script is called signal-media-exporter with hyphens. Wasn't such a great naming choice, I know...

Reply:
My bad, I just didn't pay attention. Hyphen is more standard for commands; I don't know the norm for Python modules.